contact-us/submit-a-complaint.html

```diff
@@ -12,7 +12,6 @@ <h1 class="h4">
         <ul class="list list__unstyled">
         {%- if submit_a_complaint.phone_num %}
```

Comment:
So phone_num is being updated to phone_0_num [here](https://github.com/cfpb/cfgov-refresh/blob/0351fa1b36bfd3542c4b0172de502a88f2ba4502/_lib/wordpress_contact_processor.py#L62) and then I just adjusted the markup to correspond with the shift.